docs/user-guide/cli-using-understanding-core-command-groups.md

```diff
@@ -1,205 +1,273 @@
 # Understanding core command groups
 
-Zowe CLI contains command groups that focus on specific business processes. For example, the `zos-files` command group lets you interact with mainframe data sets. This article provides a brief synopsis of the tasks that you can perform with each group. For more information, see [Displaying help](../user-guide/cli-using-displaying-help.md).
+Zowe CLI contains command groups that focus on specific business processes.
 
-The commands available in the product are organized in a hierarchical structure. Command groups (for example, `zos-files`) contain actions (for example, `create`) that let you perform actions on specific objects (for example, a specific type of data set). For each action that you perform on an object, you can specify options that affect the operation of the command.
-Zowe CLI contains the following command groups:
+Zowe CLI commands are organized in a hierarchical structure. Command groups contain actions that let you perform actions on specific objects. For each action that you perform on an object, you can specify options that affect the operation of the command.
+
+For example, the `zos-files` command group can let you perform actions on data sets such as `create`, `edit`, `rename`, and more.
+
+Review the following Zowe CLI command groups to understand the actions available to them.
 
 ## auth
 
-The auth command group lets you connect to Zowe API Mediation Layer authentication service and obtain a token, or disconnect from the authentication service and revoke the token.
+The `auth` command group lets you connect to the Zowe API Mediation Layer authentication service and obtain a token, or disconnect from the authentication service and revoke the token.
+
+:::note
 
-**Note:** For more information about `auth` syntax, actions, and options, open Zowe CLI and issue the following command:
+For more information about `auth` syntax, actions, and options, open Zowe CLI and issue the following command:
 
 ```
 zowe auth -h
 ```
+
+:::
+
 ## config
 
-The config command group lets you manage JSON projects, global configuration, and convert profiles (service profiles and base profiles) to team profiles.
+The `config` command group lets you manage JSON projects, global configuration, and convert profiles (service profiles and base profiles) to team profiles.
```

Comment:
I would change to "... lets you manage JSON project and global configurations, and ..."

Reply:
also, we should probably change "team profiles" to "team configs"